index.html

```diff
@@ -26,14 +26,11 @@
         <div class="introduction">
        <h1 class="greeting">I'm Surafel.<br>
         Glad to see you!</h1>
-        <p class="paragraph">I'm a software developer! I can help you build a product , feature or website Look through<br>
-            some of my work and experience! If you like what you see and have a project you need<br>
-            coded, don't hesitate to contact me.</p>
-        <p class="detail">I'm a software developer! I can help you<br>
-             build a product , feature or website Look<br> 
-             through some of my work and<br>
-              experience! If you like what you see and<br>
-               have a project you need coded, don't<br>
+        <p class="detail">I'm a software developer! I can help you
+             build a product , feature or website Look 
+             through some of my work and
+              experience! If you like what you see and
+               have a project you need coded, don't
```

Comment:
Congratulations on your excellent desktop design! I wanted to share a couple of friendly suggestions to enhance your portfolio project and give it an even more professional touch:

- [ ] It would be wonderful if you could align your main page section to match the Figma design provided. Currently, it appears a little misaligned.

- [ ] I noticed a slight responsiveness issue when transitioning from a mobile to a desktop screen size. To help you understand better, I've included a video below.

   [scrnli_7_14_2023_9-09-30 PM.webm](https://github.com/Surafels/Portfolio_Mobile/assets/107952724/8e50f7e8-009b-4499-a812-37926fb38d1e)